docs/execution-providers/QNN-ExecutionProvider.md

```diff
@@ -43,7 +43,7 @@ Note: Starting version 1.18.0 , you do not need to separately download and insta
     - Python 3.11.x
     - Numpy 1.25.2 or >= 1.26.4
   - Install: `pip install onnxruntime-qnn`
-  - Install nightly package `python -m pip install -i https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ORT-Nightly/pypi/simple/ort-nightly-qnn`
+  - Install nightly package `python -m pip install --pre --extra-index-url https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ORT-Nightly/pypi/simple`
```

Comment:
missed the package name in pip install command

Reply:
Fixed